microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -77,7 +81,7 @@ if (sendMessageId) {
 
 In the `onclick` event, find the current browser tab.  Then, use the `chrome.tabs.sendmessage` Extension API to send a message to that tab.
 
-In that message, you must include the URL to the image you want to display.  Also, send a unique ID to assign to the inserted image.  You could let the content insertion JavaScript generate that image ID, but for reasons that become apparent later, you'll generate that unique ID here in `popup.js`, and then pass that ID to the not-yet-created content script.
+In that message, you must include the URL to the image you want to display.  Also, make sure that you send a unique ID to assign to the inserted image.  Let the content insertion JavaScript generate that image ID in `popup.js`, and then pass that ID to the not-yet-created content script.
```

Comment:
"Let the..." is that's the approach we're NOT doing.  The original instructions direct the user to NOT let the JS generate the image ID even though they could.  The original could be clearer - trying to make too many points in one sentence.  Its structure is:

Approach 1: We're not doing this approach.  Approach 1 would be to let the content insertion JS assign the ID (we're not doing that approach).
Approach 2: We're doing this approach.  Generate the ID in popup.js, then pass the ID to the content script.

I'd definitely use two sentences and/or two list items.  I'd very likely use two list items, to line up & contrast clearly the two different approaches.  I've had success/clarity lately with naming in terms of "Approach 1" and "Approach 2".
```suggestion
In that message, you must include the URL to the image you want to display.  Also, make sure that you send a unique ID to assign to the inserted image.

To send a unique ID to assign to the image, a couple different approaches are possible:
*  Approach 1: Let the content insertion JavaScript generate that image ID.  We won't use that approach here, for reasons that become apparent later.
*  Approach 2: Generate that unique ID here in `popup.js`, and then pass that ID to the not-yet-created content script.  We'll use this approach.
```